react/components/InstallmentsContext.ts

```diff
@@ -0,0 +1,15 @@
+import { createContext } from 'react'
+
+export interface Installments {
+  Value: number
+  InterestRate: number
+  TotalValuePlusInterestRate: number
+  NumberOfInstallments: number
+  PaymentSystemName: string
+}
+
+const InstallmentsContext = createContext<
+  { installments: Installments } | undefined
+>(undefined)
+
+export default InstallmentsContext
```

Comment:
Export a Provider component and a useInstallment hook instead of the context directly